distributed/diagnostics/plugin.py

```diff
@@ -96,6 +94,14 @@ def transition(
     ) -> None:
         """Run whenever a task changes state
 
+        For a description of the transition mechanism and the available states,
+        see :ref:`Scheduler task states <scheduler-task-state>`.
+
+        Warning::
```

Comment:
This isn't rendering correctly in the built docs https://distributed--7653.org.readthedocs.build/en/7653/plugins.html

I think we want something like `.. warning::` instead to make sphinx happy (I've not built the docs locally with this suggestion though)